libs/ape-common/ape/common/prompt/prompt_base.py

```diff
@@ -225,6 +266,21 @@ def load(cls, content: str) -> "Prompt":
         config = super().load(content)
         instance = cls(**config.model_dump())
         return instance
+    
+    @classmethod
+    def load_json(cls, json: str) -> "Prompt":
```

Comment:
On second glance, it's a bit unintuitive that the argument for `load_json` is a `str`. I think it would be better if `load_json_str` loads a string and `load_json` loads a json object (dict)